bin/run-browser-test-env

```diff
@@ -1,56 +1,139 @@
 #!/bin/bash
 
-# DOC: Run the app locally using Docker for use with running browser tests.
+# DOC: Run the app locally using Docker for use with running browser tests. Run with --help to print documentation.
 
-pushd $(git rev-parse --show-toplevel)
+pushd $(git rev-parse --show-toplevel) > /dev/null
 
 set -e
 
-bin/pull-image
+source bin/lib.sh
 
-if [ -f ".secrets" ]; then
-  . .secrets
-fi
+# Default to continuous integration mode disabled.
+truth::declare_var_false continuous_integration
+
+# Default to AWS/Localstack emulator.
+cloud_provider="aws"
+emulator="localstack"
+server_url="http://localhost:6645"
+STORAGE_SERVICE_NAME="s3"
+
+#######################################
+# Ensure that only one cloud provider flag was passed tot he script.
+# Exit with error message and status code 1 if more than
+# Globals:
+#   cloud_provider
+#   already_set_cloud_provider
+# Arguments:
+#   1: the cloud provider indicated by the flag currently being processed
+#######################################
+truth::declare_var_false already_set_cloud_provider
+function ensure_only_one_cloud_provider_flag() {
+  if truth::is_enabled already_set_cloud_provider; then
+    local duplicate_providers="${cloud_provider} and ${1}"
+    echo "One cloud provider flag can be set but found both ${duplicate_providers}"
+    exit 1
+  fi
+
+  truth::enable already_set_cloud_provider
+}
 
-# Get cloud provider
-_setArgs(){
+#######################################
+# Process the script's commandline args.
+# Globals:
+#   continuous_integration
+#   already_set_cloud_provider
+#   cloud_provider
+#   emulator
+#   server_url
+#   STORAGE_SERVICE_NAME
+# Arguments:
+#   1: "$@" - full args array for the script
+#######################################
+function set_args() {
   while [ "${1:-}" != "" ]; do
     case "$1" in
+      "--help")
+        cat <<-EOF
+Usage: bin/run-browser-test-env [options]
+
+Run a local civiform environment for the browser tests to exercise.
+
+Options:
```

Comment:
I know you put it above, but I think it could be helpful here again to reiterate what the defaults are if these flags aren't set.

Reply:
done